files/anitya.spec

```diff
@@ -2,13 +2,14 @@
 %distutils.sysconfig import get_python_lib; print (get_python_lib())")}
 
 Name:           anitya
-Version:        0.10.1
+Version:        0.11.0
 Release:        1%{?dist}
 Summary:        Monitor upstream releases and announce them on fedmsg
 
 License:        GPLv2+
 URL:            https://github.com/fedora-infra/anitya/
 Source0:        %{url}/archive/%{version}/anitya-%{version}.tar.gz
+Patch0:         0001-Remove-conditional-requires.patch
```

Comment:
It's been added but not applied.

Note that for such changes you could also just use sed in the ``%prep`` section